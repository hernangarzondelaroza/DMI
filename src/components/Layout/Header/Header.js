import React from "react";
import "./Header.scss";
import { Image, Row } from "react-bootstrap";

import { DMIlogo } from "../../../utils/utils";

const header = () => {
  return (
    <Row>
      <header className="header">
        <h1>Technical Test</h1>
        <Image className="logo" src={DMIlogo} />
      </header>
    </Row>
  );
};

export default header;
