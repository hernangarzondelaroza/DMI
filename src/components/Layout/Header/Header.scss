header {
  color: white;
  background-color: black;
  padding: 1.25rem;
}

h1 {
  margin: 0 0 1rem;
}

.logo {
  max-width: 12.5rem;
}