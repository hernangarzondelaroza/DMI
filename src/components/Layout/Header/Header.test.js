import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  it('Should take a snapshot', () => {
    const component = renderer
      .create(<Header />);
    expect(component).toMatchSnapshot();
  })
})