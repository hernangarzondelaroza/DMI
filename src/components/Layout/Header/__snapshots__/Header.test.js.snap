// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should take a snapshot 1`] = `
<div
  className="row"
>
  <header
    className="header"
  >
    <h1>
      Technical Test
    </h1>
    <img
      className="logo"
      src="https://v1.nitrocdn.com/LYkBDVHVdldIrognCFlxEKUhZLCymUBO/assets/static/optimized/rev-674e467/wp-content/uploads/2020/11/white-no-sub.svg"
    />
  </header>
</div>
`;
